.github/workflows/integration.yaml

```diff
@@ -136,14 +137,18 @@ jobs:
         with:
           command: test
           args: -p ibc-integration-test --no-fail-fast --no-run
+      - name: Install cargo-nextest
+        run:
+          curl -LsSf https://get.nexte.st/latest/linux | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
```

Comment:
I want to avoid using too many random Actions, especially if the manual steps are simple. It can be difficult to verify the trustworthiness of non-official Actions.

Reply:
`taiki-e` has a good reputation but I get your point. Will revert the changes I made in the commit I just pushed, where I also use neatest for multi-chains and unit tests.